src/components/LeggTilKnapp.tsx

```diff
@@ -0,0 +1,10 @@
+import { Add } from "@navikt/ds-icons/cjs";
+import { Knapp, KnappBaseProps } from "nav-frontend-knapper";
+import React from "react";
+
+export const LeggTilKnapp = ({ children, ...rest }: KnappBaseProps) => (
```

Comment:
Hva er ...rest her?

Reply:
Det er alle KnappBaseProps sine props unntatt children (gjør destructuring og henter ut children og sender resten av propsene videre til Knapp). (se https://medium.com/@thanhbinh.tran93/props-destructuring-in-react-3bb7e8ff12e0 f.eks.)